### PR TITLE
Live migration is more complex than pg_dump/restore

### PR DESCRIPTION
**Pull request by JamesGuthrie**
# Description

Live migration is more complex than pg_dump/restore

### CODE REVIEW

File: migrate/live-migration/index.md [hunk 1]
```diff
@@ -12,7 +12,7 @@ import OpenSupportRequest from "versionContent/_partials/_migrate_open_support_r
 
 Live migration is a new migration strategy to move a large amount of data
 (100&nbsp;GB-10&nbsp;TB+) with low downtime (on the order of few minutes). It
-requires more steps to execute than a migration with downtime using
+is significantly more complex to execute than a migration with downtime using
```
Comment (VineethReddy02) [inline migrate/live-migration/index.md:15]: The introduction of a Docker image has simplified the process of live migration. The only wrinkle is stopping the CDC once its in sync.

We already have banners in our documentation, stating that live migrations are in an experimental or public preview stage. Thus, I suggest we remove the mention of “significantly complex” as discourages the user to consider this option.

**Review by VineethReddy02 — CHANGES_REQUESTED** [review]

File: migrate/live-migration/index.md (hunk 1 above)
Comment (VineethReddy02) [reply migrate/live-migration/index.md:15]: Perhaps we should eliminate the comparison with dump/restore in terms of ease of use, as the value they provide is completely different. It's an unfair comparison. 🙂

Comment (harkishen) [reply migrate/live-migration/index.md:15]: How about phrasing it gently as "can require more work to execute than a migration"?

Comment (JamesGuthrie) [reply migrate/live-migration/index.md:15]: The reality is that live migration _is_ significantly more complicated than pg_dump/restore. Users should only use it _if they really need to_. IMHO we should be clear about this.

**Review by arajkumar — APPROVED** [review]

**Review by VineethReddy02 — APPROVED** [review]

File: migrate/live-migration/index.md (hunk 1 above)
Comment (VineethReddy02) [reply migrate/live-migration/index.md:15]: I'm not completely convinced, but let's do this.

**Pull request merged**